doto-frontend/src/components/ModalContent.js

```diff
@@ -38,9 +38,13 @@ const ModalContent = props => {
     const [selectedName, setSelectedName] = useState("TASK - " + new Date());
     const [selectedDescription, setSelectedDescription] = useState("");
     const [selectedDueDate, setSelectedDueDate] = useState(new Date());
-    const [selectedDuration, setselectedDuration] = React.useState(
-        new Date("2020-08-18T01:00:00"), // duration default is 1 hour
-    );
+
+    // default duration is 1 hour
+    var initialDuration = new Date();
+    initialDuration.setHours(1);
+    initialDuration.setMinutes(0);
```

Comment:
Is the goal to add 1 hour to the current date? Because I think this will take the current date and set the time to 1am

Reply:
With the current implementation only concerning itself with hours and minutes, it seems as though the date itself is irrelevant.

Reply:
Yes, the date is irrelevent.  The way it was previously implemented, duration was passed in as minutes.  I therefore took the number of hours * 60 + minutes inserted and used that to pass into the duration.

Reply:
Oh okay, I see what you mean now